diff --git a/tests/Makefile b/tests/Makefile
index 112096d..f20d2f4 100644
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -1,9 +1,9 @@
 
-LIB_FLAGS = -larmadillo
+LIB_FLAGS += -larmadillo
 #LIB_FLAGS = -lblas -llapack 
 #LIB_FLAGS = -lopenblas -llapack 
 
-CXX_FLAGS = -std=c++11 -Wshadow -Wall -pedantic -O0
+CXX_FLAGS += -std=c++11 -Wshadow -Wall -pedantic -O0
 #CXX_FLAGS = -std=c++11 -Wshadow -Wall -pedantic -O0 -fopenmp
 #CXX_FLAGS = -std=c++11 -Wshadow -Wall -pedantic -O0 -DARMA_DONT_USE_WRAPPER
 #CXX_FLAGS = -std=c++11 -Wshadow -Wall -pedantic -O0 -fsanitize=address -fsanitize=leak -fsanitize=undefined -fsanitize=bounds -fsanitize=bounds-strict -g
