diff --git a/configure b/configure
index 9a3c977..28c2cbd 100755
--- a/configure
+++ b/configure
@@ -32929,7 +32929,7 @@ $as_echo_n "checking for FileGDBAPI... " >&6; }
     echo "#include <FileGDBAPI.h>" > testfgdb.cpp
     echo "using namespace FileGDBAPI;" >> testfgdb.cpp
     echo "int main() { Geodatabase oDB; std::wstring osStr; ::OpenGeodatabase(osStr, oDB); return 0; }" >> testfgdb.cpp
-    if test -z "`${CXX} ${CXXFLAGS} ${CPPFLAGS} testfgdb.cpp -lFileGDBAPI -lfgdbunixrtl -o testfgdb 2>&1`" ; then
+    if `${CXX} ${CXXFLAGS} ${CPPFLAGS} testfgdb.cpp -lFileGDBAPI -lfgdbunixrtl -o testfgdb`; then
         { $as_echo "$as_me:${as_lineno-$LINENO}: result: FileGDBAPI found" >&5
 $as_echo "FileGDBAPI found" >&6; }
         FGDB_ENABLED="yes"
@@ -32944,7 +32944,6 @@ $as_echo "FileGDBAPI not found." >&6; }
     fi
     rm -f testfgdb.*
     rm -f testfgdb
-
 else
 
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for libFileGDBAPI.so in in $with_fgdb/lib" >&5
