From 058e4efd61c7c9ab4efd55bd9aa37e2184654050 Mon Sep 17 00:00:00 2001
From: Zuzana Svetlikova <zsvetlik@redhat.com>
Date: Wed, 20 Sep 2017 13:05:01 +0200
Subject: [PATCH] Fix aarch64 debug

---
 deps/v8/src/v8.gyp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/deps/v8/src/v8.gyp b/deps/v8/src/v8.gyp
index c269f245aa..2ebbd393a9 100644
--- a/deps/v8/src/v8.gyp
+++ b/deps/v8/src/v8.gyp
@@ -348,7 +348,7 @@
             '<(INTERMEDIATE_DIR)/snapshot.cc',
           ],
           'variables': {
-            'mksnapshot_flags': [],
+            'mksnapshot_flags': ['--nodebug_code'],
             'conditions': [
               ['v8_random_seed!=0', {
                 'mksnapshot_flags': ['--random-seed', '<(v8_random_seed)'],
@@ -449,7 +449,7 @@
                 '<(mksnapshot_exec)',
               ],
               'variables': {
-                'mksnapshot_flags': [],
+                'mksnapshot_flags': ['--nodebug_code'],
                 'conditions': [
                   ['v8_random_seed!=0', {
                     'mksnapshot_flags': ['--random-seed', '<(v8_random_seed)'],
-- 
2.14.1

