--- dblatex-0.3.4/setup.py	2012-06-02 16:43:42.000000000 -0700
+++ dblatex-0.3.4/setup.py_disable_debian	2012-12-03 21:30:30.843851029 -0800
@@ -23,7 +23,7 @@ from distutils.command.sdist import sdis
 from distutils import log
 from subprocess import Popen, PIPE
 sys.path.append("lib")
-from contrib.debian.installer import DebianInstaller
+#from contrib.debian.installer import DebianInstaller
 
 #
 # Build the command line script
@@ -365,10 +365,10 @@ class Install(install):
             raise OSError("not found: %s" % ", ".join(mis_stys))
 
     def run(self):
-        if self.install_layout == "deb":
-            db = DebianInstaller(self)
-        else:
-            db = None
+#        if self.install_layout == "deb":
+#            db = DebianInstaller(self)
+#        else:
+        db = None
 
         if not(db) and not(self.nodeps):
             try:
