diff -up gdal-2.1.2-fedora/frmts/jpeg2000/jpeg2000_vsil_io.cpp.uchar~ gdal-2.1.2-fedora/frmts/jpeg2000/jpeg2000_vsil_io.cpp
--- gdal-2.1.2-fedora/frmts/jpeg2000/jpeg2000_vsil_io.cpp.uchar~	2016-12-01 09:14:51.768247534 -0700
+++ gdal-2.1.2-fedora/frmts/jpeg2000/jpeg2000_vsil_io.cpp	2016-12-01 09:15:14.503375681 -0700
@@ -209,7 +209,7 @@ static void JPEG2000_VSIL_jas_stream_ini
 			/* The buffer must be large enough to accommodate maximum
 			  putback. */
 			assert(bufsize > JAS_STREAM_MAXPUTBACK);
-			stream->bufbase_ = JAS_CAST(uchar *, buf);
+			stream->bufbase_ = JAS_CAST(unsigned char *, buf);
 			stream->bufsize_ = bufsize - JAS_STREAM_MAXPUTBACK;
 		}
 	} else {
