diff -ru gdal-1.10.1-fedora/swig/java/java.opt gdal-1.10.1-fedora-java/swig/java/java.opt
--- gdal-1.10.1-fedora/swig/java/java.opt	2013-08-26 16:01:11.000000000 -0400
+++ gdal-1.10.1-fedora-java/swig/java/java.opt	2016-03-15 11:25:12.223016300 -0400
@@ -1,7 +1,8 @@
 #
 # Java Stuff
 #JAVA_HOME = "C:\Program Files\Java\jdk1.6.0_16"
-JAVA_HOME = /usr/lib/jvm/java-6-openjdk/
+# Upgrade to Java8 - MCJ
+JAVA_HOME=/etc/alternatives/java_sdk/
 JAVADOC=$(JAVA_HOME)/bin/javadoc
 JAVAC=$(JAVA_HOME)/bin/javac
 JAVA=$(JAVA_HOME)/bin/java
